.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-9-release-golang-1.21-openshift-4.16
+  tag: rhel-9-release-golang-1.22-openshift-4.18
```

Comment:
Why is this using 4.18? Isn't this the 4.17 branch?

Reply:
Good catch !! 

 go1.22 image name was created for ocp18 today

Maybe Joep can help us here to be consistent ?

Reply:
@joepvd

Reply:
@hasbro17 updated 👍🏽